taca/utils/bioinfo_tab.py

```diff
@@ -143,7 +158,7 @@ def get_ss_projects(run_dir):
                 FCID_samplesheet_origin = os.path.join(run_dir,'SampleSheet.csv')
                 if not os.path.exists(FCID_samplesheet_origin):
                     logger.warn("Cannot locate the samplesheet for run {}".format(run_dir))
-                    return ['UNKNOWN']
+                    return []
```

Comment:
It was more to have a clear visual of what went wrong. @sylvinite offered to mail seqmaster instead, that would be sensible.